apps/web/src/app/agents/new/NewAgentClient.tsx

```diff
@@ -0,0 +1,663 @@
+"use client";
```

Comment:
is this really a good name for the file? From the route we know it/s agents new already.